Mlem/Views/Tabs/Settings/Components/Views/Moderation/ModerationSettingsView.swift

```diff
@@ -32,6 +33,20 @@ struct ModerationSettingsView: View {
                     "When disabled, some moderator actions will be hidden from the feed and will only be visible from when viewing a post page."
                 )
             }
+            
+            Section {
+                NavigationLink(.moderationSettings(.customizeWidgets)) {
+                    Label {
+                        Text("Customize Widgets")
+                    } icon: {
+                        if showSettingsIcons {
+                            Image(systemName: Icons.widgetWizard)
+                                .foregroundColor(.pink)
+                        }
+                    }
+                }
+            }
+            
```

Comment:
Can we add some sort of description here? Otherwise people might think it relates to post/comment widgets. Something like this?

```suggestion
            } footer: {
                Text("Customize the widgets shown under reports in the Mod Mail.")
            }
            
```